python/lance/types/image.py

```diff
@@ -361,3 +370,8 @@ def _ensure_type(images, typ):
     for im in images:
         if not isinstance(im, typ):
             raise TypeError(f"Expecting {typ} but got {type(im)}")
+
+
+def is_image(t: pa.DataType) -> bool:
```

Comment:
Maybe let's call this "is_image_type" to distinguish from checking if an object is an Image object instance?

Reply:
Done